src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,35 @@
+import { PersonRow } from '../PersonRow/PersonRow';
+import './PeopleTable.scss';
+
+type Props = {
+  people: People[]
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const peopleWithParents = people.map(child => {
+    child.mother = people.find(parent => parent.name === child.motherName)
```

Comment:
will be better to create new object with needed fields for each child
now you mutate all objects in people array